crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -392,7 +392,7 @@ impl AuthorityPerpetualTables {
         wb: &mut DBBatch,
         object: &ObjectKey,
     ) -> SuiResult<ObjectRef> {
-        wb.delete_range(
+        wb.schedule_delete_range(
```

Comment:
So, is this actually safe?

Reply:
It should be, this table is configured to not return old values post delete